src/state/store.js

```diff
@@ -5,21 +5,29 @@ import { createStore } from '../utils/create-store';
 // FIXME: take care of this dependency cycle.
 import { probeSet } from './telemetry-search'; // eslint-disable-line import/no-cycle
 
-import CONFIG from '../config/firefox-desktop';
+import productConfig from '../config/products';
+
+function getProduct() {
+  const defaultProduct = 'firefoxDesktop';
+  const mappings = {
+    firefox: 'firefoxDesktop',
+  };
+  const pathProduct = window.location.pathname.split('/')[1];
+  return pathProduct ? mappings[pathProduct] : defaultProduct;
+}
 
-// TODO: move this to the new config.js when 'product' is added.
-const DEFAULT_PROBE_PROCESS = 'content';
+const CONFIG = productConfig[getProduct()];
 
 export function getField(fieldKey) {
-  return CONFIG.dimensions[fieldKey];
+  return CONFIG.dimensions.productDimensions[fieldKey];
```

Comment:
These `getField` type functions are hard-coded to `CONFIG`, which means that whatever product is set on page load is what `CONFIG` resorts to. IT seems to me that we should be using `getProduct` in each of these functions where appropriate. It's unclear how this will work with a different product as-is.